drivers/enc28j60/enc28j60.c

```diff
@@ -85,30 +89,29 @@
 
 static void switch_bank(enc28j60_t *dev, int8_t bank)
 {
-    /* only switch bank if needed */
-    if ((bank < 0) || (dev->bank == bank)) {
+    assert(bank < 0x04);
+
+    if (bank < 0) {
```

Comment:
I'd prefer this semantic change split out (into another PR).

Reply:
I made it separate commit